install_config/uninstalling-pipelines.adoc

```diff
@@ -8,21 +8,24 @@ toc::[]
 
 Cluster administrators can uninstall the {pipelines-title} Operator by performing the following steps:
 
-. Delete the Custom Resources (CRs) that were added by default when you installed the {pipelines-title} Operator.
-. Delete the CRs of the optional components such as {tekton-hub} that depend on the Operator.
+. Delete the Custom Resources (CRs) for the optional components, `TektonHub` and `TektonChains`, if these CRs exist, and then delete the `TektonConfig` CR. 
```

Comment:
TektonChains is not optional anymore from 1.12 i think

Reply:
@piyush-garg I have replaced "optional" with "additional", is everything good now?

Reply:
so this step is valid only for optional component such as `TektonHub` and `TektonResults` and in future `Manual Approval`. This is not valid for `TektonChain` as that is not optional component and is part of `TektonConfig`